dashboard/config/routes.rb

```diff
@@ -268,8 +268,9 @@
       end
     end
 
-    # Get or create a project for the given level_id
-    get 'projects/for_level/:level_id', to: 'projects#get_or_create_for_level'
+    # Get or create a project for the given level_id. Optionally, the request
+    # can include ?script_name as a query parameter to get or create a project for the level and script.
+    get "projects/level/:level_id", to: 'projects#get_or_create_for_level'
```

Comment:
Should we use Rails' optional parameter support?

Also, should we use `script_id` rather than `script_name`?

Reply:
A small thing, and not sure whether it's even possible, but would it read more naturally if the optional `script_id` was first?

Reply:
seems like it's possible, so I'll move it!